code/game/turfs/open/floor.dm

```diff
@@ -28,12 +28,18 @@
 
 
 /turf/open/floor/Initialize(mapload)
-	if (!broken_states)
-		broken_states = string_list(list("damaged1", "damaged2", "damaged3", "damaged4", "damaged5"))
+	if (broken_states)
+		stack_trace("broken_states defined at the object level for [type], move it to setup_broken_states()")
 	else
-		broken_states = string_list(broken_states)
-	if(burnt_states)
-		burnt_states = string_list(burnt_states)
+		var/list/new_broken_states = setup_broken_states()
+		if(new_broken_states)
+			broken_states = string_list(new_broken_states)
+	if (burnt_states)
+		stack_trace("burnt_states defined at the object level for [type], move it to setup_burnt_states()")
+	else
+		var/list/new_burnt_states = setup_burnt_states()
+		if(new_burnt_states)
+			burnt_states = string_list(new_burnt_states)
```

Comment:
Call parent (. = ..) before these, instead of after.
At the start of the proc, if possible.
Does that break anything?